pyproject.toml

```diff
@@ -39,6 +38,7 @@ dev = [
     "isort",
     "ruff",
     "pytest",
+    "lxml",
```

Comment:
lxml? not xml.etree ?

Reply:
Oh just seen the commit for this, fine by me

Reply:
That’s just to test the macOS output, which contains numeric characters entities which are not supported by the Python xml parser.